config/deploy.rb

```diff
@@ -52,7 +52,7 @@
   run("ln -s #{shared_path}/nzsl_development.sqlite3 #{release_path}/db/nzsl_development.sqlite3")
 end
 
-namespace :rabid do # rubocop:disable Metrics/BlockLength
+namespace :rabid do
```

Comment:
I suspect this task isn't used any more.. we can probably remove all the capistrano remnants now that we are on heroku.

Reply:
i can clean that our in my *ahem* 10% time

Reply:
Removing in the next PR :-)

Reply:
you :rocket: !